src/lib.rs

```diff
@@ -888,7 +888,21 @@ impl<H: Helper> Editor<H> {
     /// Bind a sequence to a command.
     pub fn bind_sequence(&mut self, key_seq: KeyPress, cmd: Cmd) -> Option<Cmd> {
         if let Ok(mut bindings) = self.custom_bindings.write() {
-            bindings.insert(key_seq, cmd)
+            let key = if let KeyPress::Char(ref c) = key_seq {
+                if c.is_control() {
+                    keys::char_to_key_press(*c)
+                } else {
+                    key_seq
+                }
+            } else if let KeyPress::Ctrl(ref c) = key_seq {
+                if c.is_control() {
+                    warn!(target: "rustyline", "KeyPress::Ctrl({:?}) may not work on unix", c)
```

Comment:
In fact, based on current implementation, both unix and windows do not support such combination...

Reply:
I added these three items back in the example program and tested it and it performed as expected.
```rust
    rl.bind_sequence(KeyPress::ControlBackspace, Cmd::Kill(Movement::BackwardWord(1, Word::Big)));
    rl.bind_sequence(KeyPress::MetaBackspace, Cmd::Kill(Movement::BackwardWord(1, Word::Big)));
    rl.bind_sequence(KeyPress::ShiftBackspace, Cmd::Kill(Movement::BackwardWord(1, Word::Big)));
```
Please help me understand what you're talking about when you say, `both unix and windows do not support such a combination`

Reply:
Sorry, I mean `KeyPress::Crtl('\x08')`, for example, does not work on unix or windows...

Reply:
It's not supposed to work in this PR. That's why `ControlBackspace` was added.

Reply:
So the comment is misleading.

Reply:
I'm sorry @gwenn but I'm not following what you're saying. What comment is misleading? I was originally trying to add `KeyPress:Ctrl('\x08')` but you made it clear in this PR that it wouldn't work that way. So, I added `ControlBackspace`, `MetaBackspace`, `ShiftBackspace` in place of being able to take char literals.

Reply:
At line 899:
`KeyPress::Ctrl({:?}) may not work on unix`
should be rephrased:
`KeyPress::Ctrl({:?}) does not work`
or something like this.

Reply:
updated